community-meetings/2021-09-14.md

```diff
@@ -0,0 +1,22 @@
+# Flatcar community call Tuesday, 14th of September, 5:30 pm CEST
+
+- Zoom link (for active participation): ***[TODO: add link]***
+- Youtube live stream (for passively watching): *will be added briefly before the call starts*
+
+## Welcome
+- Brief intro / check-in of all participants in the Zoom call. Please introduce yourself and share what brings you here today.
+
+## News
+- We will discuss news and happenings in the Flatcar world, including our move out of the Kinvolk github org back into our own!
+
+## Spotlight: Equinix Metal's use of Flatcar Container Linux
+- Andy Holtzmann [@andy-v-h](https://github.com/andy-v-h) gives an introduction on EM's use of Flatcar.
+
+## Status update: ARM64
+- Progress made, remaining items, and next steps.
```

Comment:
I'm happy to speak generally to the status of arm64 server-class hardware at Equinix Metal, what's deployed and what's on the roadmap (3-5 minutes).

Reply:
Sounds good, we'll loop you in!